python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py

```diff
@@ -61,6 +58,24 @@ def is_paddle_func(func):
     return m is not None and m.__name__.startswith("paddle")
 
 
+def is_unsupported(func):
```

Comment:
Not import. It seems `is_unsupported`  is not a good name that will confuse developers, such as what is unsupported?

Reply:
Add doc for this function "Checks whether the func is supported by dygraph to static graph."